pypeit/scripts/show_2dspec.py

```diff
@@ -157,20 +161,26 @@ def main(args):
         viewer, ch = display.show_image(image, chname=chname_skysub, waveimg=spec2DObj.waveimg,
                                       bitmask=bitMask, mask=mask_in)
         if not args.removetrace and sobjs is not None:
-                show_trace(sobjs, args.det, viewer, ch)
+            show_trace(sobjs, args.det, viewer, ch)
         display.show_slits(viewer, ch, left, right, slit_ids=slid_IDs, maskdef_ids=maskdef_id)
         channel_names.append(chname_skysub)
 
 
 
     # SKRESIDS
     if 2 in show_channels:
+        # the block below is repeated because if showing this channel but not channel 1 it will crash
```

Comment:
I am not following what the purpose of the ignore_extract_mask argument is. This was admittedly added in another PR, but perhaps whomever added it can explain.

Reply:
This allows one to view the data without the EXTRACT mask being applied.